### PR TITLE
Change the answers and makes them right by Maziar

### PR DESCRIPTION
**Pull request by Maziarmajd**
# Your Details

- Your Name: Maziar Majd
- Your City: Bolton
- Your Slack Name: Maziarmajd

# Homework Details

- Module: Github
- Week: 1

### CODE REVIEW

**Review by humailhasankhan — COMMENTED** [review]
Hi Maziar, I really like how you have given detailed answers. I really like you answers. However, in order to improve, you can probably look at the styling aspects of CSS with a bit more detail if you wish to?

**Comment by Maziarmajd** [comment]
> Hi Maziar, I really like how you have given detailed answers. I really like you answers. However, in order to improve, you can probably look at the styling aspects of CSS with a bit more detail if you wish to?

Thank you dear Humail for your review. I will do that. there is a lot of things that I should learn.

File: homework.md [hunk 1]
```diff
@@ -4,38 +4,36 @@
...
-
+Hyper Text Markup Language
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
-
+CSS is the language for describing the presentation of Web pages
 ## Question 6
```
Comment (seyedsharifi12) [inline homework.md:31]: Hi @Maziarmajd you could also say something like; it is used for styling purposes to make websites more attractive.

Comment (Maziarmajd) [reply homework.md:31]: Hi Seyed, yes it is the better answer. thank you for your review

**Comment by humailhasankhan** [comment]
> > Hi Maziar, I really like how you have given detailed answers. I really like you answers. However, in order to improve, you can probably look at the styling aspects of CSS with a bit more detail if you wish to?
> 
> Thank you dear Humail for your review. I will do that. there is a lot of things that I should learn.

You're welcome Maziar. Don't worry we are all learners here. I am sure you will be helping me out with some other thing later on.

File: homework.md (hunk 1 above)
Comment (seyedsharifi12) [reply homework.md:31]: You are welcome.

**Review by SteveLeicester — COMMENTED** [review]
Thanks Maziar - some good answers.  And yes there is definitely a lot more you could say about CSS as a class member has pointed out too.

**Pull request closed without merging**